requirements.yml

```diff
@@ -0,0 +1,13 @@
+channels:
+  - conda-forge
+dependencies:
+  - python
+  - clang=16
+  - clang-format
+  - compiler-rt_linux-64
+  - gcc_linux-64=12
+  - gxx_linux-64=12
+  - llvm-openmp
+  - cmake
+  - ninja
+  - lld
```

Comment:
should we also fix lld to 16?

Reply:
I think I ran into a problem with this, at least in the beginning. But let me retry.